docs: Remove not need articles and clarify getting started (#1373)
aneoconsulting/ArmoniK: .docs/content/armonik/getting-started.md

# Getting Started with ArmoniK

> Welcome to the ArmoniK documentation! This guide will help you get started with installing, configuring, and deploying ArmoniK on various platforms. Below, you'll find links to detailed guides for each step.

Reviewer: ```suggestion
Welcome to the ArmoniK documentation! This guide will help you to get started with installing, configuring, and deploying ArmoniK on various platforms. Below, you'll find links to detailed guides for each of them.
```

> First you'll have to install ArmoniK's [Prerequisites](../getting-started/installation/prerequisites.md).

Reviewer: ```suggestion
First, you'll have to install ArmoniK's [Prerequisites](../getting-started/installation/prerequisites.md).
```

> Then you will have the choice to either deploy on your local machine, on an existing Kubernetes cluster, on AWS, or on GCP. The steps to do so are available in the following sections. The last step is to check that everything is working.

Reviewer: ```suggestion
Then, you will have the choice to either deploy on your local machine, on an existing Kubernetes cluster, on AWS, or on GCP. The steps to do so are available in the following sections. The last step is to check that everything is working.
```

## Installation on your Machine

Learn how to set up ArmoniK on your machine:

- [Local Deployment](../getting-started/installation/local.md): Deploy ArmoniK locally for development purposes.

> - [Kubernetes Deployment](../getting-started/installation/kubernetes.md): Install ArmoniK on a Kubernetes cluster.

Reviewer: This link should be removed i guess

Author: WSL and K3s installation was split since our deployment work on any Kubernetes

Author: I removed the link to Kubernetes in the getting started as we point to it in the local deployment doc

## Installation on a Kubernetes Cluster

Learn how to set up ArmoniK on Kubernetes:

- [Kubernetes Deployment](../getting-started/installation/kubernetes.md): Install ArmoniK on a Kubernetes cluster.

## Installation on AWS

Learn how to set up ArmoniK on AWS. This deploys ArmoniK and all its dependencies on AWS including Kubernetes (EKS).

- [AWS Deployment](../getting-started/installation/aws.md): Set up and deploy ArmoniK on Amazon Web Services.

## Installation on GCP

Learn how to set up ArmoniK on GCP. This deploys ArmoniK and all its dependencies on GCP including Kubernetes (GKE).

- [GCP Deployment](../getting-started/installation/gcp.md): Deploy ArmoniK on Google Cloud Platform.

## Setup Validation

Once ArmoniK is installed, learn how to execute tasks and manage workflows:

- [Check Installation](../getting-started/execution/check.md): Verify that your installation is working correctly.
- [Launch HelloWorld Sample](../getting-started/execution/how-to-launch-HelloWorld-Sample.md): Run a simple example to get started.
- [Use HTC Mock](../getting-started/execution/how-to-use-htc-mock.md): Simulate high-throughput computing scenarios.

---

> For more details, explore the specific guides linked above. Happy coding with ArmoniK!

Reviewer: ```suggestion
For more details,  we invite you to explore the specific guides linked above. Happy coding with ArmoniK!
```
